### DISCUSSION TITLE
我按照教程安装的yay有问题

### DISCUSSION THREAD

**Discussion by Nyiuu** [ArchLinuxStudio/ArchLinuxTutorial#132] [Q&A]
yay: error while loading shared libraries: libalpm.so.14: cannot open shared object file: No such file or directory

**Comment by ghost** [comment]
Your yay is broken you need to build standard Arch build process to re install it, This will force it to link against the new library version.

1st step is go in /tmp directory
and then install yay using this command

#this is the command
git clone https://aur.archlinux.org/yay.git
cd yay 
makepkg -si

#this is the best way to reuse yay package manager and best for long term uses
